integration/BUILD

```diff
@@ -12,4 +20,16 @@ py_test(
         "//:nighthawk_test_server",
         "@envoy//test/config/integration/certs",
     ],
+    # We enforce python 2, but I we may be able to stop doing that when we update
+    # bazel in CI. (Or we add python3 explicitly as a build dependency somehow).
```

Comment:
Bazel is now at 0.27 on the latest hash, there is no PY2 here, stale comment?

Reply:
Removed